### DISCUSSION TITLE
Mean peak FWHM in MS2 scans

### DISCUSSION THREAD

**Discussion by Maithy15** [vdemichev/DiaNN#250] [General]
Hi,

What do you think is a good number for mean peak FWHM in MS2 scans? This number is controlled by the cycle time right. From what I know at least 10 data points per peak is a good number. Would you say half of that is a good number in terms of FWHM? or am I interpreting this QC metric wrongly?

Thanks for the help!

**Comment by vdemichev** [comment]
Hi, we usually aim for ~3 data points per peak @ FWHM average, as reported by DIA-NN when it's run with Scan window = 0.

Best,
Vadim
